x/gamm/types/msgs.go

```diff
@@ -177,3 +178,21 @@ func (m MsgExitSwapShareAmountIn) GetSigners() []sdk.AccAddress {
 	}
 	return []sdk.AccAddress{sender}
 }
+
+var _ sdk.Msg = &MsgUpdateSwapFee{}
+
+func (m MsgUpdateSwapFee) Route() string { return RouterKey }
+func (m MsgUpdateSwapFee) Type() string  { return TypeMsgUpdateSwapfee }
+func (m MsgUpdateSwapFee) ValidateBasic() error {
+	return nil // TODO
+}
+func (m MsgUpdateSwapFee) GetSignBytes() []byte {
+	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&m))
+}
+func (m MsgUpdateSwapFee) GetSigners() []sdk.AccAddress {
+	sender, err := sdk.AccAddressFromBech32(m.Sender)
```

Comment:
`SwapFeeGovernor` is multisig address?

Reply:
It can be.  Typically it will be a governance account though (see the work started here:  https://github.com/sikkatech/cosmos-sdk/pull/66)